src/js/lib.js

```diff
@@ -373,7 +359,7 @@ vjs.TEST_VID = vjs.createEl('video');
  * @constant
  * @private
  */
-vjs.USER_AGENT = navigator.userAgent;
+var USER_AGENT = navigator.userAgent;
```

Comment:
I think (for now) this needs to stay a var because of the JSHint issues @heff was running into. Perhaps switching to eslint will fix this?